pyproject.toml

```
@@ -29,37 +29,37 @@ ignore_missing_imports = true
 
 [tool.poetry.dependencies]
 python = "^3.8.10"
-pydantic = "^2.7,<3.0"
-jsonschema = "^4.21.1"
-sentencepiece = "0.2.0"
-typing-extensions = "^4.11.0"
-tiktoken = "^0.7.0"
-pillow = "^10.3.0"
-requests = "^2.0.0"
+pydantic = ">=2.7,<3.0"
```

```suggestion
pydantic = "^2.7"
```

Think for Pydantic we should leave between 2 and 3 no?